src/sonic-config-engine/minigraph.py

```diff
@@ -212,7 +212,8 @@ def parse_dpg(dpg, hname):
             for i, member in enumerate(vmbr_list):
                 vmbr_list[i] = port_alias_map[member]
             vlan_attributes = {'name': vintfname, 'members': vmbr_list, 'vlanid': vlanid}
-            vlans[vintfname] = vlan_attributes
```

Comment:
maybe it is better to add vlanid in the swss as the vlan name could be some meaningful string? like server v.s. bmc